pimpmycause/profiles/forms.py

```diff
@@ -29,4 +29,19 @@ class Meta:
 
 class S3DirectUploadForm(forms.Form):
 
-    image = forms.URLField(widget=S3DirectWidget(dest='user-profile-images'))
+    #image = forms.URLField(widget=S3DirectWidget(dest='user-profile-images'))
+    images = forms.URLField(widget=S3DirectWidget(
+        dest='user-profile-images',
+        html=(
+            '<div class="s3direct" data-policy-url="{policy_url}">'
+            '  <a class="file-link" target="_blank" href="{file_url}">{file_name}</a>'
+            '  <a class="file-remove" href="#remove">Remove</a>'
+            '  <input class="file-url" type="hidden" value="{file_url}" id="{element_id}" name="{name}" />'
+            '  <input class="file-dest" type="hidden" value="{dest}">'
+            '  <input class="file-input js-input-file inputfile" type="file" id="file" name="file"/>'
+            '  <label for="file" class="button">Choose an image</label>'
```

Comment:
There is an 'Images:' text just above the 'Choose an image' button which I cannot see in the code.. 
![image](https://cloud.githubusercontent.com/assets/13818956/18974761/38d66570-869d-11e6-83e4-ad3731dcde31.png)